### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 6

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7516]
**Preferred term label:**
Abnormal Vaginal Bleeding

**Synonyms**

**Definition (free text, please give PubMed ID)**
PubMed ID: 21250125, Abnormal vaginal bleeding occurs between menstrual periods, after sex, or after menopause. Menstrual periods that are heavier or last longer than usual or last more than seven days also are considered abnormal.

**Parent term (use hpo.jax.org/app)**
Abnormal bleeding HP:0001892

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
n/a

**Your nano-attribution (ORCID)**
n/a

**Comment by pnrobinson** [comment]
added as HP_0034263
